src/customizations/volto/components/theme/View/LinkView.jsx

```diff
@@ -13,20 +13,20 @@ import config from '@plone/volto/registry';
 
 const LinkView = ({ token, content }) => {
   const history = useHistory();
-  const userIsSpidUser = useSelector(
-    (state) => state.users.user.roles.length === 0,
+  const isReader = useSelector((state) =>
```

Comment:
Va benissimo il fix, mi piace. Per eccesso di pignoleria però cambierei questo nome, suggerisce che l'utente abbia il ruolo Reader quando in realtà non è quello che indica.

Reply:
```suggestion
  const isCitizen = useSelector((state) =>
```
poi metterei un commento che dice che al momento il cittadino è un utente senza ruoli, in futuro potrebbe essere un utente con il ruolo 'Citizen'

Reply:
@pnicolli aggiornato